composer.json

```
@@ -20,7 +20,7 @@
         "codeception/codeception": "^5.0.8",
         "codeception/lib-xml": "^1.0",
         "justinrainbow/json-schema": "^5.2.9 || ^6",
-        "softcreatr/jsonpath": "^0.8 || ^0.9 || ^0.10"
+        "softcreatr/jsonpath": "^0.8 || ^0.9 || ^0.10 || ^0.11.0"
```

```suggestion
        "softcreatr/jsonpath": "^0.8 || ^0.9 || ^0.10 || ^0.11"
```

@npwvestjens Please change the version constraint for consistency

---

Nice, I missed that somehow.